src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/VisitsByDateService.kt

```diff
@@ -49,9 +49,8 @@ class VisitsByDateService(
       val visitorCount = visit.visitors?.size ?: 0
       val visitTimeSlot = SessionTimeSlotDto(visit.startTimestamp.toLocalTime(), visit.endTimestamp.toLocalTime())
       try {
-        prisonerSearchClient.getPrisonerById(visit.prisonerId)?.let { prisoner ->
```

Comment:
prisonerSearchClient.getPrisonerById cannot be null anymore hence removed the ?.let